menu.kando.Kando.metainfo.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>menu.kando.Kando</id>
+
+    <name>Kando</name>
+    <summary>The Cross-Platform Pie Menu.</summary>
+
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>MIT</project_license>
+
+    <supports>
+        <control>pointing</control>
+        <control>keyboard</control>
+        <control>touch</control>
+        <control>gamepad</control>
+        <control>tablet</control>
+    </supports>
+
+    <description>
+        <p>
+            Kando is a cross-platform pie menu for your desktop. It offers an unconventional, fast,
+            highly efficient, and fun way of interacting with your computer! You can use Kando to
+            launch applications, simulate keyboard shortcuts, open files, and much more.
+        </p>
+        <p>
+            Kando is designed to be used with 🖱️ mouse, 🖊️ stylus, 👆 touch, or 🎮 controller input.
+            If you have both hands at your keyboard most of the time, Kando is maybe not the right
+            tool for you.
+        </p>
+    </description>
+
+    <url type="homepage">https://kando.menu/</url>
+    <url type="bugtracker">https://github.com/kando-menu/kando/issues</url>
+    <url type="donation">https://schneegans.github.io/sponsors/</url>
+    <url type="translate">https://hosted.weblate.org/engage/kando/</url>
+    <url type="help">https://discord.gg/hZwbVSDkhy</url>
+
+    <launchable type="desktop-id">menu.kando.Kando.desktop</launchable>
+
+    <developer id="io.github.schneegans">
+        <name>Simon Schneegans</name>
+    </developer>
+
+    <content_rating type="oars-1.1" />
+
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/kando-menu/flatpak/refs/heads/main/screenshots/screenshot01.jpeg</image>
+        </screenshot>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/kando-menu/flatpak/refs/heads/main/screenshots/screenshot02.jpeg</image>
+        </screenshot>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/kando-menu/flatpak/refs/heads/main/screenshots/screenshot03.jpeg</image>
+        </screenshot>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/kando-menu/flatpak/refs/heads/main/screenshots/screenshot04.jpeg</image>
```

Comment:
screenshots links should be tied to a commit or a tag not to main branch `refs/heads/main`

Reply:
Okay, done.